src/renderer/store/TimelineSpace/Contents/Search/Tag.ts

```diff
@@ -0,0 +1,47 @@
+import Mastodon, { Tag, Results } from 'megalodon'
+import { Module, MutationTree, ActionTree } from 'vuex'
+import { RootState } from '@/store'
+
```

Comment:
__[ESLint]__ Error - More than 1 blank line not allowed. (no-multiple-empty-lines) ([link](https://sider.review/gh/repos/124234495/pulls/877#issue-3979074))

<sub>posted by Sider</sub>